dask_geomodeling/geometry/field_operations.py

```diff
@@ -216,87 +204,97 @@ def process(source, other):
 
 
 class Add(BaseFieldOperation):
-    """
-    Addition of series and other, element-wise.
+    """ Addition of constant value or seriesblock to a different seriesblock (element-wise).
+    
+    Supply a seriesblock and either a second seriesBlock or a constant value to be added to its data. The data are added.
 
-    :type source: SeriesBlock
-    :type other: SeriesBlock, float
-
-    See also:
-      https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.add.html
+    Args:
+      a (input seriesBlock 1): The seriesblock which is used in the calculation. Datatype: seriesBlock
+      b (value to add): Either a second seriesblock or a constant value which is added to the first one. Datatype: seriesBlock or float
+    
+    Returns:
+      seriesBlock where the values are summed.
     """
 
     process = staticmethod(operator.add)
 
 
 class Subtract(BaseFieldOperation):
-    """
-    Subtraction of series and other, element-wise.
+    """ Subtract scalar or seriesBlock from another seriesBlock.
+    
+    Supply a seriesBlock and either a second seriesBlock or a constant value to be subtracted from its data. 
 
-    :type source: SeriesBlock
-    :type other: SeriesBlock, float
-
-    See also:
-      https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.subtract.html
+    Args:
+      a (Input seriesBlock 1): The seriesblock which is used in the calculation. Datatype: seriesBlock
+      b (value to subtract): Either a second seriesblock or a constant value which is subtracted from the first one. Datatype: seriesBlock or float
+      
+    Returns:
+      seriesBlock where the values are subtracted.
```

Comment:
Note that if you want to subtract a seriesblock from a constant value (like `4 - series`, you have to do `Add(Multiply(series, -1), 4)`